core-java/src/main/java/io/github/boykaframework/parser/ExcelParser.java

```diff
@@ -137,12 +137,15 @@ private <T> List<T> getDataFromFile (final List<Object[]> data, final Constructo
                     final var value = rowData[col];
                     methodName = format ("set{0}", capitalize (header));
                     if (!isNull (value)) {
+                        if (value instanceof Boolean) {
```

Comment:
Please remove this if condition, because it is for debug purpose.

Reply:
Resolved..